spec/indexers/identifiable_indexer_spec.rb

```diff
@@ -148,5 +146,32 @@
         expect(doc).to match a_hash_including('metadata_source_ssi' => 'DOR')
       end
     end
+
+    context 'with identification sub-schema' do
```

Comment:
Should this be `with no identification sub-schema`?

Reply:
Yes, thank you.